### PR TITLE
Added missed "template" settings

### PR DESCRIPTION
**Pull request by omurbekjk**
### Description
[Describe what this change achieves]
This change fixes below error when creating index template shown in step 2.
```
{
  "error" : {
    "root_cause" : [
      {
        "type" : "x_content_parse_exception",
        "reason" : "[3:3] [index_template] unknown field [settings]"
      }
    ],
    "type" : "x_content_parse_exception",
    "reason" : "[3:3] [index_template] unknown field [settings]"
  },
  "status" : 400
}
```

 
### Issues Resolved
[List any issues this PR will resolve]
 
### Check List
- [x] Commits are signed per the DCO using --signoff 

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

File: _im-plugin/ism/policies.md [hunk 1]
```diff
@@ -558,9 +558,11 @@ The following sample template policy is for a rollover use case.
    PUT _index_template/ism_rollover
    {
      "index_patterns": ["log*"],
-     "settings": {
-       "plugins.index_state_management.rollover_alias": "log"
-     }
+     "template": {
+      "settings": {
+       "opendistro.index_state_management.rollover_alias": "log"
+      }
```
Comment (ashwinkumar12345) [inline _im-plugin/ism/policies.md:564]: Do we need to replace "opendistro" with "plugins"?

File: _im-plugin/ism/policies.md [hunk 2]
```diff
@@ -586,6 +588,12 @@ The following sample template policy is for a rollover use case.
    }
    ```
 
+5. Verify if the policy is attached to the `log-000001` index:
+
+   ```json
+   GET _opendistro/_ism/explain/log-000001?pretty
```
Comment (ashwinkumar12345) [inline _im-plugin/ism/policies.md:594]: Replace "opendistro" with "plugins"?

**Review by ashwinkumar12345 — APPROVED** [review]

File: _im-plugin/ism/policies.md (hunk 1 above)
Comment (omurbekjk) [reply _im-plugin/ism/policies.md:564]: verified both of the requests work.
GET _opendistro/_ism/explain/log-000001?pretty

GET _plugins/_ism/explain/log-000001?pretty

But to be consistent I changed opendistro to plugins

**Comment by aetter** [comment]
Many thanks @omurbekjk!

**Pull request merged**